scripts/tests/test_ecs_helpers.py

```diff
@@ -56,6 +56,29 @@ def test_sorted_by_multiple_keys(self):
         result = ecs_helpers.dict_sorted_by_keys(dict, ['group', 'name'])
         self.assertEqual(result, expected)
 
+    def test_merge_dicts(self):
+        a = {
+            'cloud': {'group': 2, 'name': 'cloud'},
+            'agent': {'group': 2, 'name': 'agent'},
+        }
+        b = {'base': {'group': 1, 'name': 'base'}}
+
+        result = ecs_helpers.safe_merge_dicts(a, b)
+
+        self.assertEquals(result,
+            {
+                'cloud': {'group': 2, 'name': 'cloud'},
+                'agent': {'group': 2, 'name': 'agent'},
+                'base': {'group': 1, 'name': 'base'}
+            })
```

Comment:
Ah sorry! Seems like Python's autopep8 (code formatting) isn't happy with the nesting here.

If you run `make check` locally, autopep8 will modify the file in place, to be formatted correctly. Looks like it's aligning the opening curly with the function's opening parens.

Reply:
Nice catch! Sorry about that! I think I've got it fixed.